src/cmd/flux-jobs.py

```diff
@@ -290,8 +348,10 @@ def parse_args():
         "-o",
         "--format",
         type=str,
+        default="default",
         metavar="FORMAT",
-        help="Specify output format using Python's string format syntax",
+        help="Specify output format by name (use 'help' to get a list of names)"
+        + ", or using Python's string format syntax",
```

Comment:
"or using" -> "or use"

Reply:
I think "or using" is correct here (I think the real problem is the unnecessary comma), because "or using Python's string format syntax" still follows "Specify output format..." as the other half of the "or", but I agree it is awkward. Maybe the order should be reversed here so the parenthetical comes at the end, i.e.

> Specify output format using Python's string format syntax or a defined format by name (use 'help' to get a list of names).